gherkin/php/tests/ParserException/UnexpectedTokenExceptionTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cucumber\Gherkin\ParserException;
+
+use Cucumber\Gherkin\GherkinLine;
+use Cucumber\Gherkin\Location;
+use Cucumber\Gherkin\Token;
+use PHPUnit\Framework\TestCase;
+
+final class UnexpectedTokenExceptionTest extends TestCase
+{
+    public function testMessageSpecifiesWhichTokenWasUnexpected(): void
+    {
+        $line = $this->createMock(GherkinLine::class);
+        $line->method('getLineText')->willReturn('Baz');
+
+        $exception = new UnexpectedTokenException(
+            new Token($line, new Location(10, 10)),
+            ['Foo', 'Bar'],
+            'stateComment'
```

Comment:
I recommend the following setting for PHP-CS-Fixer:
```php
        'trailing_comma_in_multiline' => [
            'after_heredoc' => true,
            'elements' => [
                'arrays',
                'arguments',
                'parameters',
            ],
        ],
```

for adding trailing commas where possible

Reply:
Thanks, I've been trying to do that so it'll be good to use the rule 👍